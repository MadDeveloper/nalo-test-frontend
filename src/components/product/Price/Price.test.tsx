import renderer from "react-test-renderer"
import { Price } from "."

describe("Components > Price", () => {
  it("should match snapshot", () => {
    const tree = renderer.create(<Price amount={10} />).toJSON()

    expect(tree).toMatchSnapshot()
  })
})
