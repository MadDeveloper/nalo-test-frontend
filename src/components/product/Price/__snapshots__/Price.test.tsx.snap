// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components > Price should match snapshot 1`] = `
<span
  className="root"
>
  10.00
   ETH
</span>
`;
